src/Model/LoginManager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Model;
+
+class LoginManager extends AbstractManager
+{
+    public const TABLE = 'Login';
+
+    public function checkUser(array $user)
+    {
+        $statement = $this->pdo->prepare("SELECT * FROM " . self::TABLE . " WHERE `user` = :user");
+        $statement->bindValue('user', $user['user'], \PDO::PARAM_STR);
+        $statement->execute();
+        return $statement->fetch();
+    }
+
+    public function checkPass(array $pass)
+    {
+        $statement = $this->pdo->prepare("SELECT * FROM " . self::TABLE . " WHERE `pass` = :pass");
+        $statement->bindValue('pass', password_hash($pass['pass'], PASSWORD_BCRYPT), \PDO::PARAM_STR);
+        $statement->execute();
+        return $statement->fetch();
+    }
```

Comment:
No you don't check for login then for the password. 
If you do that you check if the login exists, if the password exists but not if this is the right password for the right login.

You have to check them both at the same time with the same query.

Reply:
Stop the login, dev, put him in another branch of dev. DragAndDrop pull skill from a database and not in from scratch.